includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -2767,44 +2923,102 @@ function( $id ) use ( $dom ) {
 					}
 				}
 			}
+
+			$pending_stylesheet['stylesheet']    = implode( '', $stylesheet_parts );
 			$pending_stylesheet['original_size'] = $original_size;
+			$pending_stylesheet['included']      = null; // To be determined below.
+			$pending_stylesheet['size']          = strlen( $pending_stylesheet['stylesheet'] ); // @todo Should this use mb_strlen()?
```

Comment:
The mbstring extension might not always be available. We would need to add a requirement to `amp.php`

Reply:
Yeah, I clarified the comment to indicate that this is something to explore in the future. There are other instances of `strlen()` being used above.

Part of it is we need to clarify if the AMP validator uses _characters_ or bytes when calculating the limit. Using byte count is safer in this way. Actually, it would seem to be the latter since the spec has `max_bytes` not `max_chars`. So I think I just answered my own question 😄